src/main/java/frc/robot/commands/RotationCommand.java

```diff
@@ -25,6 +27,7 @@ public RotationCommand(SpinnerSubsystem spinner) {
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
+    Constants.m_RobotState.setState(States.ROTATION);
```

Comment:
Drive state isn't set if currently in Rotation - We never exit the Rotation state - Make sure to have an "exit" state ie. Go back to idle after a state has been completed so that other states can take priority